### PR TITLE
Add link to integration docs from service control

### LINKED ISSUE #8278: Add URL Link to Documentation in the Service Call Tools [/developer-tools/service]
I think it would be a nice addition to add a link to the documentation.

Right under the FILL EXAMPLE DATA (which is also a really great feature).

What do you think?

For example; when I am at the service window `/developer-tools/service` and I enter `media_player.volume_mute` it should give me a link to these integration docs:

https://www.home-assistant.io/integrations/media_player/

Preferably under the "Fill Example" data option.

I think this can be done programmatically and would make independent learning much easier, especially for new users. It might even avoid silly questions on the Discord or community forum.

Shitty text example

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Adds a help icon with a link tot he integration documentation for the selected service domain.

**UI mode:**
image

**YAML mode:**
image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes https://github.com/home-assistant/frontend/issues/8278
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by spacegaier** [comment]
@bramkragten Is this PR something that you think makes sense? If yes, I will resolve the merge conflicts, otherwise of course not 😄.

**Comment by bramkragten** [comment]
With the new UI I don't think it is really useful anymore? The docs will be based on the same data as we show in the UI in the future.

**Comment by spacegaier** [comment]
> 
> 
> With the new UI I don't think it is really useful anymore? The docs will be based on the same data as we show in the UI in the future.

In the docs there can be more than just the service data, such as additional context info, examples, etc. Even though that is then not strictly tied to the service testing, it might make it easy to navigate to look up some additional details.

**Comment by bramkragten** [comment]
I'll ask advise from our docs maistro, @frenck 

Personally I don't really see the use...

**Comment by SeanPM5** [comment]
I could see this being useful in certain scenarios. An example would be a TTS service that supports multiple voices/languages, the available voices are usually listed in the docs.

**Comment by frenck** [comment]
Got pinged by @spacegaier on this matter...
I don't have an opinion on this matter.

I hope we can generate documentation for services based on the service descriptions (so in the end, it should be the same). 🤷

File: src/translations/en.json [hunk 1]
```diff
@@ -3433,6 +3433,7 @@
             "column_description": "Description",
             "column_example": "Example",
             "fill_example_data": "Fill Example Data",
+            "help": "Integration help",
```
Comment (bramkragten) [inline src/translations/en.json:3436]:
```suggestion
            "help": "Integration documentation",
```

File: src/components/ha-service-control.ts [hunk 2]
```diff
@@ -174,7 +176,29 @@ export class HaServiceControl extends LitElement {
...
+      <div class="description">
+        <p>${serviceData?.description}</p>
+        ${this.value?.service
+          ? html` <a
+              href="${documentationUrl(
+                this.hass,
+                "/integrations/" + computeDomain(this.value?.service)
+              )}"
+              title="${this.hass.localize(
+                "ui.panel.developer-tools.tabs.services.help"
```
Comment (bramkragten) [inline src/components/ha-service-control.ts:188]: This component is used in more panel then just the `developer-tools` panel, the translation should be in the component section.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**